example/example_server/Makefile

```diff
@@ -0,0 +1,34 @@
+REBAR := rebar3
+
+.PHONY: all
+all: compile
+
+compile:
+	$(REBAR) compile
+
+.PHONY: clean
+clean: distclean
+
+.PHONY: distclean
+distclean:
+	@rm -rf _build erl_crash.dump rebar3.crashdump rebar.lock
+
+.PHONY: xref
+xref:
+	$(REBAR) xref
+
+.PHONY: eunit
+eunit: compile
+	$(REBAR) eunit verbose=truen
+
+.PHONY: ct
+ct: compile
+	$(REBAR) as test ct -v
```

Comment:
Is `as test` really needed in the context of `ct` command? I think rebar3 already sets the profile to test in it.

Reply:
It's a shame that I don't understand the operation mode of this file very well. Could you please teach me the correct form of example makefile?

Reply:
You can just drop `as test` in this line.

rebar3 actually replaces most of the features of makefile, so it can be shortened to something like this:
https://github.com/k32/typerefl/blob/master/Makefile

Reply:
Thank you!